lib/mysql_rewinder/cleaner.rb

```diff
@@ -20,8 +21,11 @@ def clean(tables:)
       target_tables = (tables - @except_tables) & all_tables
       return if target_tables.empty?
 
-      @client.execute("SET FOREIGN_KEY_CHECKS = 0;")
-      @client.execute(target_tables.map { |table| "DELETE FROM #{table}" }.join(';'))
+      disable_foreign_key_checks = "SET FOREIGN_KEY_CHECKS = 0;"
+      delete_sql = target_tables.map { |table| "DELETE FROM #{table}" }.join(';')
+
+      log_and_execute(disable_foreign_key_checks)
+      log_and_execute(delete_sql)
```

Comment:
These variables are not seemed to be necessary.

```suggestion
      log_and_execute("SET FOREIGN_KEY_CHECKS = 0;")
      log_and_execute(target_tables.map { |table| "DELETE FROM #{table}" }.join(';'))
```